src/App.tsx

```diff
@@ -1,61 +1,129 @@
 import './App.scss';
+import { useState } from 'react';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
+import { TodoList } from './components/TodoList';
+import { User } from './components/types/User';
+import { Todo } from './components/types/Todo';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+const newTodos: Todo[] = todosFromServer.map(todo => ({
+  ...todo,
+  user: usersFromServer.find(user => user.id === todo.userId) || null,
+}));
+
+export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>(newTodos);
+  const [title, setTitle] = useState('');
+  const [titleTouched, setTitleTouched] = useState(false);
+  const [titleError, setTitleError] = useState(false);
+  const [user, setUser] = useState('');
+  const [userTouched, setUserTouched] = useState(false);
+  const [userError, setUserError] = useState(false);
+  const [users] = useState<User[]>(usersFromServer);
```

Comment:
You are using `useState` to store the `users` array, but since you are not updating this state, you can directly use the `usersFromServer` array without wrapping it in `useState`. This will simplify your code and reduce unnecessary state management.